addons/hr_holidays/report/hr_leave_report.py

```diff
@@ -51,9 +47,9 @@ def init(self):
                 leaves.employee_id as employee_id, leaves.name as name,
                 leaves.number_of_days as number_of_days, leaves.leave_type as leave_type,
                 leaves.number_of_hours as number_of_hours,
-                leaves.category_id as category_id, leaves.department_id as department_id,
+                leaves.department_id as department_id,
```

Comment:
You should then remove the `category_id` field definition (line 25). Needs upgrade script.